rohmu/object_storage/azure.py

```diff
@@ -199,6 +214,7 @@ def delete_key(self, key):
             return blob_client.delete_blob()
         except azure.core.exceptions.ResourceNotFoundError as ex:  # pylint: disable=no-member
             raise FileNotFoundFromStorageError(path) from ex
+        self.notifier.object_deleted(key)
```

Comment:
This would not get called, as the `return` is in `try:` block

Reply:
oops, good caught :)